envisage/plugins/ipython_kernel/tests/test_ipython_kernel_plugin.py

```diff
@@ -35,9 +33,6 @@
                      "skipping tests that require the ipykernel package")
 class TestIPythonKernelPlugin(unittest.TestCase):
 
-    def tearDown(self):
-        IPKernelApp.clear_instance()
```

Comment:
just to confirm, this is useless because `IPKernelApp.clear_instance` doesn't do anything?

Reply:
Right, the `stop` method for the plugin already clears the instance, so if that `stop` method has been called, this line doesn't do anything. (And if it isn't called, we've got bigger issues and this line isn't going to help much.)